keras_nlp/models/bert/bert_tasks.py

```diff
@@ -29,7 +29,7 @@
     Examples:
     ```python
     # Randomly initialized BERT encoder
-    model = keras_nlp.models.BertCustom(
```

Comment:
If you want you can put the input data first like you requested in the other docstrings.

Reply:
Done!